doc/howto/usage/cluster/fluid_cluster_train_en.md

```diff
@@ -0,0 +1,138 @@
+# Fluid Distributed Training
+
+## Introduction
+
+In this article, we'll explain how to config and run distributed training job with PaddlePaddle Fluid in a bare metal cluster.
+
+## Preparations
+
+### Get your cluster ready
+
+Prepare your computer node in the cluster. Nodes in this cluster can be any spec that runs PaddlePaddle, and with a unique IP address assigned to it. Make sure they can talk to each other.
```

Comment:
node -> **nodes**

Reply:
can be any spec -> can be **of any specification**

Reply:
I think it would be better to use the word **communicate** instead of talk as this is a formal document.

Reply:
will update